modules/adhashBidAdapter.js

```diff
@@ -120,7 +121,7 @@ function brandSafety(badWords, maxScore) {
       .replaceAll(/\s\s+/g, ' ')
       .toLowerCase()
       .trim();
-    const content = window.top.document.body.innerText.toLowerCase();
+    const content = window.top.document.body.outerText.toLowerCase();
```

Comment:
does textContent not work? that's what we used in 9.0 branch

Reply:
https://kellegous.com/j/2013/01/26/layout-performance/ outerText just as bad

Reply:
Understood. We've updated the code. Thanks for your comment.